### ISSUE TITLE
Help Installing Dimp50

### ISSUE THREAD

**Issue by yogeshv4** [visionml/pytracking#321]
Hi,

I need some help installing pytracking because i have followed each step thoroughly and even used the install.sh to try load this program on both linux and windows 10. Both of this operating system the pytracking does not work. 

Is it possible for anyone that have this program up and running can me the exact version of the :
pytorch
torchvision
cudatoolkit

And did you guys install the submodules as well of the pytracking.

Main Issue i am getting are:

- AssertionError: Torch not compiled with CUDA enabled
- No GPU Available(eventhough both my machine have GPU)
- No module called torch.fx 
- cant find from torch int_classes, string_classes from torch._six ( i worked around that by setting int_classes = int and string_classes=str)
- Sometime i get Local.py is not SET

Could anyone please help

**Comment by Ahsanr312** [comment]
Yes, I have been using the repository on linux, it works absolutely fine. 
Python 3.6, 3.7
Pytorch 1.8 and its compatible CUDA and CUDNN versions

Steps to follow
1. Clone the repository
2. Clone the sub modules
3. Then from installation.sh start your installation process one by one via apt-get and pip3
4. Install each library one by one and solve issues if you encounter any while installing any library. For me, pip install spatial-correlation-sampler was unable to solve but as they have mentioned its only for KYS tracker so you can skip it as well. 

I hope it helps.

**Comment by Ahsanr312** [comment]
@yogeshv4 Were you able to make it working? If not then let me know we will discuss and sort it out.

**Comment by mj8ac** [comment]
> Yes, I have been using the repository on linux, it works absolutely fine. My hardware is Jetson TX2. Python 3.6, 3.7 Pytorch 1.8 and its compatible CUDA and CUDNN versions
> 
> Steps to follow
> 
> 1. Clone the repository
> 2. Clone the sub modules
> 3. Then from installation.sh start your installation process one by one via apt-get and pip3
> 4. Install each library one by one and solve issues if you encounter any while installing any library. For me, pip install spatial-correlation-sampler was unable to solve but as they have mentioned its only for KYS tracker so you can skip it as well.
> 
> I hope it helps.

Are you able to run Dimp-50 on your TX-2? If so what sort of FPS do you get? 

We are currently trying to implement Dimp-50 in TensorRT but will probably need to write a custom plugin. Before that we want to be able to parse the network via pytorch to onnx however onnx is also complaining that it does not support PrRoiPooling. I think the only way round that is to submit a pr to PyTorch maintainers for this layer. 

Thanks, 
Jonny
